src/components/GroceryItem.svelte

```diff
@@ -0,0 +1,155 @@
+<script>
+  export let item;
+  export let removeItem;
+  export let updateItem;
+  export let editItem;
+  export let itemId;
+  export let isEditing = false;
+
+  export let inputEl;
+  let trashIconImage = "./images/trash.svg";
+  let editIconImage = "./images/edit-icon.svg";
+  //export let inputEl = item.name;
+</script>
+
+<li class="item">
+  {#if isEditing}
+    <input class="input-update" type="text" bind:value={inputEl} />
```

Comment:
with `export let inputEl` you are overcomplicating. you don't need to export that as a prop, it can be a simple variable without the export.

What you are looking for is the `value` HTML property to set a initial default value. 

```
<input value={item.name} class="input-update" type="text" bind:value={inputEl} />
```